packages/nextjs/components/example-ui/ContractData.tsx

```diff
@@ -1,4 +1,5 @@
 import { useEffect, useRef, useState } from "react";
+import { gql, useQuery } from "@apollo/client";
```

Comment:
I'd use [URQL](https://github.com/urql-graphql/urql/tree/main/packages/react-urql) for querying The Graph/GraphQL in Next.js. There were some weird problems with SSR.